Markup fixes and better semantics

jhung opened the pull request.

jhung committed:
NO-JIRA: Work in progress. Styling and wording.

jhung committed:
NOJIRA: Added role status to description.

jhung committed:
NOJIRA: Added aria live region.

jhung committed:
NOJIRA: Added fixes for various ATs.

See “Mar 11, 2016” at
https://wiki.fluidproject.org/display/fluid/PhET+Notes for testing
results based on this branch.

jhung committed:
NOJIRA: Tweaked markup.

jobara commented on sim.js:
```diff
@@ -19,10 +19,13 @@ var init = function () {
     alertElm.setAttribute("role", "alert");
+    alertElm.setAttribute("class", "alert");
+    alertElm.setAttribute("aria-atomic", "false");
+    alertElm.setAttribute("aria-live", "assertive");
```
Are you sure this is needed? I was reading the MDN page about aria-live and it indicated that this could cause double speaking. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

jhung committed:
NOJIRA: Removed aria markup injection

Remove uneeded aria markup done via Javascript. This also fixes alerts
not being reported in Safari.

jhung committed:
FLUID-5897: Added two different implementations of charge reporting.
Added two alternative charge reporting methods: paragraph status, and
text field. Index.html allows you to choose from the 3 implementations.

jhung committed:
NO-JIRA: Really rough hack to get status paragraph usable for testing
- changes the content of alert and status messages.
- removed charge description from scene description.

jhung committed:
NO-JIRA: Added some descriptive text explaining the demo.

jhung committed:
Changed "negative charges" to "electrons" and other wording tweaks.

jhung committed:
Create README.md
